app/views/docs/query.phtml

```diff
@@ -0,0 +1,276 @@
+        <h3>Flutter</h3>
+        <div class="ide" data-lang="javascript" data-lang-label="Flutter SDK">
+        <pre class="line-numbers"><code class="prism language-dart" data-prism>import 'package:appwrite/appwrite.dart';
+String queryString = Query.lesserEqual(attribute, value);
+print(queryString);</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Android</h3>
+        <div class="ide" data-lang="kotlin" data-lang-label="Android SDK">
+            <pre class="line-numbers"><code class="prism language-kotlin" data-prism>import 'package:appwrite/appwrite.dart';
+String queryString = Query.lesserEqual(attribute, value)
+print(queryString)</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>iOS</h3>
+        <div class="ide" data-lang="swift" data-lang-label="Apple SDK">
+            <pre class="line-numbers"><code class="prism language-swift" data-prism>import Appwrite
+let queryString = Query.lesserEqual(attribute, value)
+print(queryString)</code></pre>
+        </div>
+    </li>
+</ul>
+
+<h2><a href="/docs/query#greater-than" id="greater-than">Greater Than</a></h2>
+<p>
+Returns results where the specified <b>attribute</b> is greater than <b>value</b>. Can be applied to strings where comparison is based on alphabetical order.
+</p>
+<ul class="phases clear" data-ui-phases>
+    <li>
+        <h3>Web</h3>
+        <div class="ide" data-lang="javascript" data-lang-label="Web SDK">
+            <pre class="line-numbers"><code class="prism language-javascript" data-prism>import { Query } from "appwrite";
+const queryString = Query.greaterEqual(attribute, value);
+console.log(queryString);</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Flutter</h3>
+        <div class="ide" data-lang="javascript" data-lang-label="Flutter SDK">
+        <pre class="line-numbers"><code class="prism language-dart" data-prism>import 'package:appwrite/appwrite.dart';
+String queryString = Query.greaterEqual(attribute, value);
+print(queryString);</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Android</h3>
+        <div class="ide" data-lang="kotlin" data-lang-label="Android SDK">
+            <pre class="line-numbers"><code class="prism language-kotlin" data-prism>import 'package:appwrite/appwrite.dart';
+String queryString = Query.greaterEqual(attribute, value)
+print(queryString)</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>iOS</h3>
+        <div class="ide" data-lang="swift" data-lang-label="Apple SDK">
+            <pre class="line-numbers"><code class="prism language-swift" data-prism>import Appwrite
+let queryString = Query.greaterEqual(attribute, value)
+print(queryString)</code></pre>
+        </div>
+    </li>
+</ul>
+
+<h2><a href="/docs/query#greater-than" id="greater-than">Greater Than or Equal To</a></h2>
+<p>
+Returns results where the specified <b>attribute</b> is greater than or equal to <b>value</b>. Can be applied to strings where comparison is based on alphabetical order.
```

Comment:
In all these examples , we should mention that value can either be a single value like Avengers or a list of values like ['Avengers', 'Titanic', 'Iron Man']

Reference : https://github.com/appwrite/sdk-for-web/blob/master/src/sdk.ts#L3748-L3767

You can use a numerical example to make it more clear